### PR TITLE
Update Journal management doc and deprecated properties

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/operation/Journal.md [hunk 1]
```diff
@@ -104,11 +104,9 @@ If this is not set, clients will look for masters using the hostnames from `allu
 and the master rpc port (Default:`19998`).
 
 ### Advanced configuration
+
 * `alluxio.master.embedded.journal.transport.request.timeout.ms`: The duration after which embedded journal masters will timeout messages sent between each other.
  Lower values might cause leadership instability when network is slow. Default: `5s`.
-* `alluxio.master.embedded.journal.appender.batch.size`: Size of a single internal knowledge batch sent from leader to secondary masters.
```
Comment (LuQQiu) [inline docs/en/operation/Journal.md:109]: @bf8086 are the following properties need to be added as advanced configuration:
```
MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT
MASTER_EMBEDDED_JOURNAL_WRITE_LOCAL_FIRST_ENABLED 
MASTER_EMBEDDED_JOURNAL_SNAPSHOT_REPLICATION_CHUNK_SIZE
```

File: docs/en/operation/Journal.md [hunk 2]
```diff
@@ -343,6 +340,10 @@ Therefore, Alluxio primary master will not create checkpoints by default.
 Restarting the current primary master to transfer the leadership to another running master periodically
 can help avoiding primary master journal logs from growing unbounded when Alluxio is running in HA mode.
 
+Starting from Alluxio 2.4.0, Alluxio primary master can download checkpoints from standby masters
+and install in its journal folder in embedded journal HA mode. By doing so, Alluxio primary master 
+can truncate its journal size without causing temporary service unavailability.
```
Comment (calvinjia) [inline docs/en/operation/Journal.md:345]: Could you clarify if the user needs to do anything on their end or if this is an automatic optimization?

**Review by calvinjia — CHANGES_REQUESTED** [review]

File: docs/en/operation/Journal.md (hunk 2 above)
Comment (LuQQiu) [reply docs/en/operation/Journal.md]: Modify the words, please take another look, thanks!

File: docs/en/operation/Journal.md (hunk 1 above)
Comment (bf8086) [reply docs/en/operation/Journal.md:109]: Yes.

**Comment by LuQQiu** [comment]
@calvinjia PTAL, thanks!

**Review by calvinjia — APPROVED** [review]

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**